documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd

```diff
@@ -70,6 +71,8 @@ outdir<- "/fs/data2/output/PEcAn_99000000001/out/99000000001"
 start.year<-1997
 end.year<-2004
 site.id<-772
+File_Path<-"/fs/data3/tmccabe/AMF_US-NR1_BASE_HH_9-1.csv" #This is a path to where the observations are stored. If you have your own observations, change to cusom file path. 
```

Comment:
Got that you're moving these things earlier, but as noted previously these need to be the default paths -- where PEcAn will put the data automatically and where users will see on the VMs --  not your personal directories